sympy/utilities/tests/test_debug_logs.py

```diff
@@ -0,0 +1,41 @@
+"""
+A way to check if debugging works.
+"""
+from __future__ import print_function
+import subprocess
+import sys
+import os
+import inspect
+from os.path import abspath, dirname, join, normpath
+from contextlib import contextmanager
+
+@contextmanager
+def environ(env):
+    """
+    set environment variables
+    :param env: environment variable to modify
+    """
+    original_environ = os.environ.copy()
+    os.environ.update(env)
+    yield
+    os.environ = original_environ # reset the environment variable to prevent side effect
+
+def test_debug_logs():
+    my_filename = abspath(inspect.getfile(inspect.currentframe()))
```

Comment:
It would be better to use the `tempfile` module if we needed to create files but we don't since we can't use `-c`:
```
$ python -c 'from sympy import *; x = Symbol("x"); print(integrate(exp(x)*cos(x), x))'
exp(x)*sin(x)/2 + exp(x)*cos(x)/2
```